src/MAUI/Maui.Samples/Samples/Layers/DisplayPointsUsingClusteringFeatureReduction/DisplayPointsUsingClusteringFeatureReduction.xaml.cs

```diff
@@ -0,0 +1,79 @@
+        instructions: "Pan and zoom the map to view how clustering is dynamically updated. Toggle clustering off to view the original point features that make up the clustered elements. When clustering is On, you can tap on a clustered geoelement to view aggregated information and summary statistics for that cluster. When clustering is toggled off and you tap on the original feature you get access to information about individual power plant features.",
+        tags: new[] { "aggregate", "bin", "cluster", "group", "merge", "normalize", "reduce", "summarize" })]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData()]
+    public partial class DisplayPointsUsingClusteringFeatureReduction
+    {
+        private FeatureLayer _layer;
+
+        public DisplayPointsUsingClusteringFeatureReduction()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            // Get the power plants web map from the default portal.
+            var portal = await ArcGISPortal.CreateAsync();
+            PortalItem portalItem = await PortalItem.CreateAsync(portal, "8916d50c44c746c1aafae001552bad23");
+
+            // Create a new map from the web map.
+            MyMapView.Map = new Map(portalItem);
+
+            // Get the power plant feature layer once the map has finished loading.
+            await MyMapView.Map.LoadAsync();
+            _layer = (FeatureLayer)MyMapView.Map.OperationalLayers.First();
+
+            // Hide and nullify an opened popup when user taps screen.
+            PopupBackground.GestureRecognizers.Add(new TapGestureRecognizer
+            {
+                Command = new Command(() =>
+                {
+                    PopupBackground.IsVisible = false;
+                    PopupViewer.Popup = null;
+                })
+            });
+        }
+
+        private async void MyMapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.Maui.GeoViewInputEventArgs e)
+        {
+            // Identify the tapped observation.
+            IdentifyLayerResult results = await MyMapView.IdentifyLayerAsync(_layer, e.Position, 3, true);
+
+            // Return if no observations were found.
+            if (results.Popups.Count == 0) return;
+
+            // Set the popup and make it visible.
+            PopupViewer.Popup = results.Popups.FirstOrDefault();
+            PopupBackground.IsVisible = true;
+        }
+
+        // Enable clustering feature reduction if the checkbox has been checked, disable otherwise.
+        private void CheckBox_CheckChanged(object sender, CheckedChangedEventArgs e)
+        {
+            // This event is raised when sample is initially loaded when layer is null.
+            if (_layer == null) return;
+
+            _layer.FeatureReduction.IsEnabled = (bool)(sender as CheckBox).IsChecked;
```

Comment:
```suggestion
            _layer.FeatureReduction.IsEnabled = (sender as CheckBox).IsChecked;
```

Cast doesn't seem to be required.

Reply:
This is not required for MAUI, but is still required for WinUI and WPF.